lambdas/llama-2-13b-chat-llm/template.yml

```diff
@@ -0,0 +1,112 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: QnABot on AWS LLM Plugin for Llama-2-13b-chat
```

Comment:
deployed using SageMaker JumpStart (add link)

Reply:
Addressed in the 2nd commit.